application/jaxrs-rabbitmq-quarkus/src/main/java/webapp/tier/service/RabbitmqConsumer.java

```diff
@@ -0,0 +1,42 @@
+package webapp.tier.service;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.DefaultConsumer;
+import com.rabbitmq.client.Envelope;
+
+import webapp.tier.bean.MsgBean;
+import webapp.tier.service.socket.RabbitmqSocket;
+import webapp.tier.util.MsgUtils;
+
+public class RabbitmqConsumer extends DefaultConsumer {
+
+	private static final Logger LOG = Logger.getLogger(RabbitmqConsumer.class.getSimpleName());
+	private static String splitkey = ConfigProvider.getConfig().getValue("rabbitmq.split.key", String.class);
+	RabbitmqSocket rmqsock = new RabbitmqSocket();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5968983)